tiledb/ml/readers/tensorflow.py

```diff
@@ -4,7 +4,7 @@
 from tensorflow.python.data.ops.dataset_ops import FlatMapDataset
 
 
-class TensorflowTileDBDenseDataset(FlatMapDataset):
+class KerasTensorflowTileDBDenseDataset(FlatMapDataset):
```

Comment:
I think that finally we don't need Keras here. We only use the tensorflow data api, so it's not needed. Apologies for suggesting it. Please revert.